agent/engine/docker_task_engine_linux.go

```diff
@@ -39,3 +44,76 @@ func (engine *DockerTaskEngine) updateTaskENIDependencies(task *apitask.Task) {
 func (engine *DockerTaskEngine) invokePluginsForContainer(task *apitask.Task, container *apicontainer.Container) error {
 	return nil
 }
+
+func (engine *DockerTaskEngine) watchAppNetImage(ctx context.Context, filepath string) {
+	watcher, err := fsnotify.NewWatcher()
+	if err != nil {
+		seelog.Errorf("error adding %s to fsnotify watcher, error: %v", filepath, err)
+	}
+	err = watcher.Add(filepath)
+	if err != nil {
+		seelog.Errorf("failed to initialize fsnotify NewWatcher, error: %v", err)
```

Comment:
Just noticed that the two error messages should probably be swapped :)

Reply:
Thanks for catching that! just updated:)